gprofiler/merge.py

```diff
@@ -94,7 +100,11 @@ def merge_perfs(perf_all: Iterable[Mapping[str, str]], process_perfs: Mapping[in
         if process_perf_count > 0:
             ratio = perf_all_count / process_perf_count
             for stack, count in process_stacks.items():
-                full_stack = ";".join([process_names[pid], stack])
-                new_samples[full_stack] += round(count * ratio)
-
-    return "\n".join((f"{stack} {count}" for stack, count in new_samples.items()))
+                stack_line = ";".join([process_names[pid], stack])
+                stack_line_with_container_name = docker_client.get_stack_line_with_container_name(stack_line, pid)
+                new_samples[stack_line_with_container_name] += round(count * ratio)
+    container_names = docker_client.get_container_names()
+    docker_client.reset_cache()
+    output = [f"#{json.dumps({'containers': container_names})}"]
```

Comment:
as @ezean21 asked, can you please also include the `hostname` in the awesome metadata dict?
thanks!